src/main/java/de/hpi/bpt/argos/api/response/ResponseFactoryImpl.java

```diff
@@ -123,13 +174,188 @@ public String getEventsForProduct(long productId, long eventTypeId, int eventInd
 	public String getEvent(long eventId) {
 		Event event = entityManager.getEvent(eventId);
 		if (event == null) {
-			halt(ResponseFactory.getHttpNotFoundCode(), "Event not found");
+			halt(ResponseFactory.HTTP_NOT_FOUND_CODE, "Event not found");
 		}
 		JsonObject jsonEvent = getEvent(event);
 		return jsonEvent.toString();
 	}
 
 	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void createEventType(String requestBody) {
+		try {
+			JsonObject jsonBody = jsonParser.parse(requestBody).getAsJsonObject();
+
+			String eventQuery = jsonBody.get(JSON_EVENT_QUERY_ATTRIBUTE).getAsString();
+			JsonObject jsonEventType = jsonBody.get(JSON_EVENT_TYPE_ATTRIBUTE).getAsJsonObject();
+
+			if (eventQuery == null || eventQuery.length() == 0) {
+				halt(ResponseFactory.HTTP_ERROR_CODE, "no event query given in body");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "no event query given in body" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)